.gitignore

```
@@ -14,4 +14,5 @@ cygnus-twitter/bin
 .classpath
 .project
 .factorypath
-*/.settings
\ No newline at end of file
+*/.settings
+.*
```

```suggestion
*/.settings
.*
```

as I understand that files like `.gitignore` itself should be included in the repo :)

---

Fixed in 704e06e